lib/Elastica/Scroll.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Elastica;
+
+/**
+ * Scroll Iterator
+ *
+ * @author Manuel Andreo Garcia <andreo.garcia@gmail.com>
+ *
+ * @link http://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-scroll.html
+ */
+class Scroll implements \Iterator
+{
+    /**
+     * @var string
+     */
+    public $expiryTime;
+
+    /**
+     * @var Search
+     */
+    protected $_search;
+
+    /**
+     * @var null|string
+     */
+    protected $_nextScrollId = null;
+
+    /**
+     * @var null|ResultSet
+     */
+    protected $_currentResultSet = null;
+
+    /**
+     * 0: scroll<br>
+     * 1: scroll id<br>
+     * 2: search type.
+     *
+     * @var array
+     */
+    protected $_options = array(null, null, null);
```

Comment:
I was wondering if we should be using Value Objects for these pieces of config. But I haven't seen them implemented anywhere else in the library, so I would assume that arrays are the conventions in this case. Isn't that the case @ruflin?